cli/config/configuration-acrn.toml.in

```diff
@@ -235,3 +235,10 @@ sandbox_cgroup_only=@DEFSANDBOXCGROUPONLY@
 # Supported experimental features:
 # (default: [])
 experimental=@DEFAULTEXPFEATURES@
+
+# If enabled, containers are allowed to join the pid namespace of the agent
+# when the env variable AGENT_PIDNS is set for a container.
+# Use this with caution and only when required, as this option allows the container
+# to access the agent process. It is recommended to enable this option 
+# only in debug scenarios and with containers with lowered priveleges.
```

Comment:
Should there be an upper bound on the total duration where this feature is enabled ?
So that in case the user forgets to unset it, the feature would be turned off after certain time period.